cpp/src/aztec/honk/pcs/ipa/ipa.test.cpp

```diff
@@ -4,50 +4,48 @@
 #include "./polynomials/polynomial_arithmetic.hpp"
 #include "./polynomials/polynomial.hpp"
 #include <ecc/curves/bn254/fq12.hpp>
+#include "honk/pcs/commitment_key.hpp"
+#include "honk/pcs/commitment_key.test.hpp"
 using namespace barretenberg;
+namespace honk::pcs::ipa {
 
-TEST(honk_commitment_scheme, ipa_commit)
-{
-    constexpr size_t n = 1024;
-    std::vector<barretenberg::fr> scalars(n);
-    std::vector<barretenberg::g1::affine_element> points(n);
+template <class Params> class IpaCommitmentTest : public CommitmentTest<Params> {
+    using Fr = typename Params::Fr;
+    using element = typename Params::Commitment;
+    using affine_element = typename Params::C;
+    using CK = typename Params::CK;
+    using VK = typename Params::VK;
+    using Polynomial = barretenberg::Polynomial<Fr>;
+};
 
-    for (size_t i = 0; i < n; i++) {
-        scalars[i] = barretenberg::fr::random_element();
-        points[i] = barretenberg::g1::affine_element(barretenberg::g1::element::random_element());
-    }
+TYPED_TEST_SUITE(IpaCommitmentTest, IpaCommitmentSchemeParams);
 
-    barretenberg::g1::element expected = points[0] * scalars[0];
+TYPED_TEST(IpaCommitmentTest, commit)
+{
+    constexpr size_t n = 128;
+    auto poly = this->random_polynomial(n);
+    auto CK = this->ck();
```

Comment:
`CK` is used for the type of the commitment key, so it would be better to call this `ck` or `key`.

Reply:
Thanks for pointing this out. Removed this and directly using `srs` now e.g. `auto srs_elements = this->ck()->srs.get_monomial_points();`